common/source/docs/common-pixracer-overview.rst

```diff
@@ -73,6 +73,7 @@ Specifications
    -  TELEM1/TELEM2
    -  Wifi serial
    -  FrSky Telemetry serial
+   Note: The FrSky telemetry port is implemented using a 74LVC2G86 NOR gate as an inverter to both the RX and TX lines
```

Comment:
```suggestion
   Note: The FrSky telemetry port is implemented using a 74LVC2G86 NOR gate as a programmable inverter to both the RX and TX lines
```

Reply:
actually this opens a small can of worms...can we chat when you get a chance?